WalletWasabi.Fluent/Views/MainView.axaml

```diff
@@ -47,7 +47,7 @@
               </Style>
             </Panel.Styles>
             <Border Background="{DynamicResource RegionColor}" BoxShadow="{DynamicResource BoxShadow}" IsHitTestVisible="False" />
-            <ContentControl Content="{Binding MainScreen.CurrentPage}" Margin="20 36 20 20" />
+            <ContentControl ClipToBounds="False" Content="{Binding MainScreen.CurrentPage}" Margin="20 36 20 20" />
```

Comment:
A bit looks weird for me that we have a full separator at one place then another without edges at another place.
I personally prefer the separators without edges.

![image](https://user-images.githubusercontent.com/16364053/117954687-8c09bc00-b317-11eb-93dd-148d6c5bf31c.png)

Reply:
im going to defer this until navbar re-work has been done.